docs/source/concepts.rst

```diff
@@ -186,10 +192,26 @@ the list of PyPI packages manually using the :code:`pip_packages` option:
       def predict(self, df):
           return self.artifacts.model.predict(df)
 
-Note that although this supports the use of remote Git URLs, any use of Pip package options like :code:`-i` or :code:`-f` is not supported. If you'd like to use those features, you can define your own :code:`requirements.txt` file and pass it using the :code:`requirements_txt_file` option by doing :code:`@bentoml.env(requirements_txt_file='./requirements.txt')`.
+.. code-block:: python
+  :caption: Specifying PyPI packages through the `requirements_txt_file` option
+
+  @bentoml.env(
+    requirements_txt_file="./requirements.txt"
+  )
+  class ExamplePredictionService(bentoml.BentoService):
+
+      @bentoml.api(input=DataframeInput(), batch=True)
+      def predict(self, df):
+          return self.artifacts.model.predict(df)
+
+Note that although this supports the use of remote Git URLs, any use of Pip package options 
```

Comment:
This line is describing the `pip_packages` option, maybe move it to before the L196 where it starts talking about `requirements_txt_file`?

Reply:
Or change it to:

```
This `requirements.txt` file should be compatible with the latest version of `pip`, and supports all the pip package install options including  :code:`--index-url` and :code:`--find-links`.
```